frappe/custom/doctype/doctype_layout/doctype_layout.js

```diff
@@ -15,6 +15,7 @@ frappe.ui.form.on('DocType Layout', {
 					for (let f of frappe.get_doc('DocType', frm.doc.document_type).fields) {
 						frm.add_child('fields', { fieldname: f.fieldname, label: f.label });
 					}
+					frm.refresh_fields();
```

Comment:
This is still not a better fix, it still shows the error even if the fields were added. Don't fix this problem with this PR as it is already been worked on by PR https://github.com/frappe/frappe/pull/16418
```suggestion
```